src/cmd/list.go

```diff
@@ -22,7 +22,7 @@ func init() {
 }
 
 func isJsonOutput() bool {
-	return listOutputType == "json"
+	return listOutputType == "json" || getOutputType == "json"
```

Comment:
This might break other `get *` commands.  We might need to not use `isJsonOutput` and instead move this logic into `writeOutput`